core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -1927,6 +1927,10 @@ private boolean isIntervalHourAndSecond(SqlCall call) {
       case INTERVAL_DAY_SECOND:
       case INTERVAL_DAY_MINUTE:
       case INTERVAL_DAY_HOUR:
+      case INTERVAL_DAY:
+      case INTERVAL_HOUR:
```

Comment:
can we check for INTERVAL_YEAR and INTERVAL_SECOND also
and update the test case accordingly.

Reply:
Done